apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -90,6 +92,15 @@ export default class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _ensureEnvironment(): void {
+    /* tslint:disable-next-line:no-string-literal */
+    let environmentPath: string = process.env['PATH'];
+    environmentPath = path.join(this.rushConfiguration.commonTempFolder, 'node_modules', '.bin') +
```

Comment:
That is the limitation of locally installed npm package. We should provide the capability of allowing people to run the locally installed program if they want, but we are not going to solve the npm version problem for them. 

---
In reply to: [119754113](https://github.com/Microsoft/web-build-tools/pull/218#discussion_r119754113) [](ancestors = 119754113,119753539,119751422)